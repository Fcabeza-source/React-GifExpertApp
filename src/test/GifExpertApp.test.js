import React from 'react';
import { shallow } from 'enzyme';
import'@testing-library/jest-dom';
import GifExpertApp from '../GifExpertApp';



describe('Pruebas sobre <GifExpertApp />', () => {
    
    const wrapper = shallow(<GifExpertApp />);

    test('Debe de funcionar correctamente ', () => {

        expect(wrapper).toMatchSnapshot();

    })
    
})
