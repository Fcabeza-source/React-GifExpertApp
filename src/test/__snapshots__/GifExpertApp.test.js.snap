// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre <GifExpertApp /> Debe de funcionar correctamente  1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="One Piece"
      key="One Piece"
    />
  </ol>
</Fragment>
`;
